packages/extension-chains/src/index.ts

```diff
@@ -17,6 +17,33 @@ const definitions = new Map<string, MetadataDef>(
   CENNZNetChain.map((def) => [def.genesisHash, def])
 );
 
+export function getLatestMetaFromServer(genesisHashExpected: string) {
```

Comment:
nit: pls add some return types to this function

Reply:
added